test/withMessageTest.js

```diff
@@ -0,0 +1,75 @@
+var chai = require('chai');
+var expect = chai.expect;
+var request = require('supertest');
+
+var app;
+var errorMessage = 'Default error message';
+var mustBeTwoDigitsMessage = 'testparam must have two digits';
+var mustBeIntegerMessage = 'testparam must be an integer';
+
+function validation(req, res) {
+  req.checkParams('testparam', errorMessage)
+    .notEmpty()
```

Comment:
Maybe have another comment here that says `// with default message` like `isInt()` below, just so it's super clear.